src/features/TodoForm.jsx

```diff
@@ -0,0 +1,38 @@
+//import React from 'react';
+import { useRef } from 'react';
+import {useState} from 'react';
+import TextInputWithLabel from '../shared/TextInputWithLabel';
+
+
+
+function TodoForm({onAddTodo}) {
+    const [workingTodoTitle, setWorkingTodo] = useState('');
+    const todoTitleInput= useRef(''); 
+
+    function handleAddTodo(event){
+        event.preventDefault();
+        const title = workingTodoTitle.trim();
+        if(title === '')
+          return;
+        onAddTodo(workingTodoTitle);
+        setWorkingTodo('')
```

Comment:
If you are already using `useRef()`, you might as well add re-focus on your input after adding:
```js
if (todoTitleInput.current) {
  todoTitleInput.current.focus();
}
```